decidim-debates/app/cells/decidim/debates/debate_card_metadata_cell.rb

```diff
@@ -9,25 +9,16 @@ class DebateCardMetadataCell < Decidim::CardMetadataCell
 
       alias debate model
 
-      delegate :type_of_meeting, :start_time, :end_time, :category, :withdrawn?, to: :debate
+      delegate :type_of_meeting, :start_time, :end_time, :taxonomies, :withdrawn?, to: :debate
```

Comment:
The `DebateCardMetadataCell` is actually using the `taxonomy_items` defined in `CardMetadataCell`. In that method, you are invoking `resource.taxonomies`. 

```suggestion
      delegate :type_of_meeting, :start_time, :end_time, :withdrawn?, to: :debate
```